appium-dotnet-driver/appium-dotnet-driver.csproj

```diff
@@ -67,6 +67,7 @@
     <Compile Include="Appium\Enums\MobileBrowserType.cs" />
     <Compile Include="Appium\Enums\MobileCapabilityType.cs" />
     <Compile Include="Appium\Android\Enums\ConnectionType.cs" />
+    <Compile Include="Appium\Enums\WindowsKeyCodes.cs" />
```

Comment:
I think it is better to create a folder in `Appium\Windows\` and name it `Enums` and then put the file to this folder